src/lamplib/__main__.py

```diff
@@ -68,8 +81,8 @@ def main():
         tasks.clean(context, compile_env)
     else:
         tasks.compile_all(context, compile_env)
-
         if args.subcommand == 'install':
+            check_venv(args)
```

Comment:
hm why not just always `check_venv` at the top of the method?

Reply:
Running cmake tests and such don't require the venv so I didn't want to make people enter the venv. This was before it started being done automatically so now I'm fine with always doing it